vtex-apps/store-image
# Add ImageSlider component and interface

#### What did you change? \*

Create a new `<ImageSlider />` component and new `image-slider` interface that is customizable from the Site Editor.

#### Why? \*

To enable the user to easily create and control a slider of images both from the `*.json` file and from our Site Editor.

#### How to test it? \*

Go to https://imageslider--storecomponents.myvtex.com/admin/cms/site-editor/about-us
and you should see `Image Slider` on the right-side navigation. Select it and you should be able to play around with the Image Slider through the Site Editor =D

#### Related to / Depends on?

PR #1 and PR #2 came before this.
Also, this should be released after `vtex.slider-layout@0.5.0` to support editing from Site Editor.

#### Types of changes \*
